hai/pipe_pump.py

```diff
@@ -37,7 +37,7 @@ def pump(self, timeout=0):
         with self.selector_lock:
             if not self.selector:  # pragma: no cover
                 return
-            for (key, event) in self.selector.select(timeout=timeout):
+            for (key, _event) in self.selector.select(timeout=timeout):
```

Comment:
If not used I'd expect `for (key, _) ...` but apparently flake8 suggest just a leading underscore?

Reply:
Yep. I like this way more since you still have a context of what the other thing in the tuple is, even if you don't use it.

This is, coincidentally, the way you mark variables unused in [Rust](https://stackoverflow.com/a/48361729/51685) too.